projects/visual-regression/src/styles.scss

```diff
@@ -1 +1,27 @@
-@import '~@terminus/design-tokens/css/library-design-tokens.css';/* You can add global styles to this file, and also import other style files */
+@import '~@terminus/ui/helpers';
+@import '~@terminus/ui/terminus-ui.css';
+@import '~@terminus/design-tokens/css/library-design-tokens.css';
+
+html {
+  background-color: color(pure);
+  font-family: $type__family--base;
```

Comment:
Don't forget you can use our CSS custom property tokens for this too ;p (def not a needed change)